sdk/src/azure/core/az_mqtt5_rpc_client_codec.c

```diff
@@ -8,18 +8,18 @@
 
 #include <azure/core/_az_cfg.h>
 
-AZ_NODISCARD az_mqtt5_rpc_client_options az_mqtt5_rpc_client_options_default()
+AZ_NODISCARD az_mqtt5_rpc_client_codec_options az_mqtt5_rpc_client_codec_options_default()
 {
-  return (az_mqtt5_rpc_client_options){ .subscribe_timeout_in_seconds
-                                        = AZ_MQTT5_RPC_DEFAULT_TIMEOUT_SECONDS,
-                                        .publish_timeout_in_seconds
-                                        = AZ_MQTT5_RPC_DEFAULT_TIMEOUT_SECONDS,
-                                        .subscription_topic_format = AZ_SPAN_EMPTY,
-                                        .request_topic_format = AZ_SPAN_EMPTY };
+  return (az_mqtt5_rpc_client_codec_options){ .subscribe_timeout_in_seconds
```

Comment:
Nit: I believe all these options are the rpc_client_options not codec options (the codec can't really do much regarding the timeouts).

The `topic_format`s though make sense part of the codec.